### ISSUE TITLE
Stop passing in the approval_prompt=force query param to the initial sign in page

### ISSUE THREAD

**Issue by JeffTax** [getsentry/sentry#73879]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

Login using Google to Sentry using http://sentry.io

### Expected Result

After one approval, subsequent approvals aren't needed

### Actual Result

Approvals are required every time.

### Product Area

Sign In

**Comment by leedongwei** [comment]
It might be a default thing from the library that we're using. Agreed that removing it would be a small win.

**Comment by anonymous1-wq** [comment]
Recent ticket related to this feedback: https://sentry.zendesk.com/agent/tickets/134536

**Comment by JeffTax** [comment]
What do you all think?

**Comment by leedongwei** [comment]
@JeffTax Seems like the param was an intentional choice (see [code](https://github.com/getsentry/sentry/blob/e288751c73cc3e87da2e446a8d2178f293ddd8d2/src/sentry/auth/providers/google/provider.py#L28-L30)) so we'd want to take some time to carefully dig into it.

We're doing a larger refactor of the authentication experience so this will be addressed during that effort.

**Comment by leonard84** [comment]
I hope this gets fixed soon. It isn't very pleasant for every Google SSO user on every login.

**Comment by leonard84** [comment]
It is really disappointing that this seemingly quick fix has still not been implemented. It adds needless friction to the login process and trains people to be less careful with approvals since it becomes the norm rather than the exception.

> We're doing a larger refactor of the authentication experience so this will be addressed during that effort.

When is that refactoring happening?

**Comment by JeffTax** [comment]
Yeah, I hate to pile on, but I think tracking this issue has taken more effort than fixing it.

**Comment by leedongwei** [comment]
Hello folks, I'm resource constrained at the moment, so it's hard to get movement on authentication work. I'll schedule time next week for me to figure out what they were trying to do with `prompt=force`over [here](https://github.com/getsentry/sentry/blob/e288751c73cc3e87da2e446a8d2178f293ddd8d2/src/sentry/auth/providers/google/provider.py#L28-L30) and see if I can put in the fix.

**Comment by leonard84** [comment]
The fix is simple: just remove the parameter, and everyone will be happy.

**Comment by JeffTax** [comment]
@leedongwei any updates?.
